console/console-init/ui/src/modules/address/containers/AddressDefinitionContainer/AddressDefinitionContainer.tsx

```diff
@@ -151,7 +151,8 @@ export const AddressDefinitionContainer: React.FunctionComponent<IAddressDefinit
       }
       if (
         type?.toLowerCase() === "subscription" ||
```

Comment:
Please use enum type `AddressTypes` for values "queue", "topic" and "subscription"